tests/servers/servers.go

```diff
@@ -87,7 +69,15 @@ func (server *HTTPServer) Contents(w http.ResponseWriter, r *http.Request) {
 }
 
 func (server *HTTPServer) Certificates(w http.ResponseWriter, r *http.Request) {
-	w.Write(servedPublicKey)
+	w.Write(fixtures.PublicKey)
+}
+
+func (server *HTTPServer) CABundle(w http.ResponseWriter, r *http.Request) {
+	w.Write(fixtures.CABundle)
+}
+
+func (server *HTTPServer) InvalidCABundle(w http.ResponseWriter, r *http.Request) {
+	w.Write(fixtures.PublicKey)
```

Comment:
This is the same as the `Certificates` endpoint, so why not just use that rather than creating a new endpoint?

Reply:
Yeah, makes sense to me. Let me change the code.